pkg/controllers/plugin/helm_controller.go

```diff
@@ -250,6 +256,7 @@ func (r *HelmReconciler) reconcileHelmRelease(
 	if err := helm.InstallOrUpgradeHelmChartFromPlugin(ctx, r.Client, restClientGetter, pluginDefinition, plugin); err != nil {
 		reconcileFailedCondition.Status = metav1.ConditionTrue
 		reconcileFailedCondition.Message = "Helm install/upgrade failed: " + err.Error()
+		metrics.UpdateMetrics(plugin, metrics.MetricResultError, metrics.MetricReasonUninstallHelmFailed)
```

Comment:
This should be `Upgrade` not `Uninstall`

Reply:
The called function also updates the metric. Please ensure the metric is not updated twice.